smarts/sstudio/genhistories.py

```diff
@@ -168,7 +168,7 @@ def create_output(self):
         self._log.debug("shifting sim_times..")
         mcur = dbconxn.cursor()
         mcur.execute(
-            "UPDATE Trajectory SET sim_time = sim_time - (SELECT min(sim_time) FROM Trajectory)"
+            "UPDATE Trajectory SET sim_time = round(sim_time - (SELECT min(sim_time) FROM Trajectory), 3)"
```

Comment:
I think that the digit param could at minimum be a descriptive class constant like `TIME_DECIMAL_ACCURACY` since it has a large impact on the behaviour of this class.